src/TraceEvent/Symbols/SymbolReader.cs

```diff
@@ -979,7 +979,11 @@ internal bool GetPhysicalFileFromServer(string serverPath, string pdbIndexPath,
                         {
                             m_log.WriteLine("FindSymbolFilePath: In task, sending HTTP request {0}", fullUri);
 
-                            var responseTask = HttpClient.GetAsync(fullUri, HttpCompletionOption.ResponseHeadersRead);
+                            // Create request with MSFZ support
```

Comment:
Let's modify this comment to be "Tell the symbol server that we support MSFZ symbols."